primitives/rpc/debug/src/api/single.rs

```diff
@@ -101,26 +102,26 @@ pub struct RawStepLog {
 
 #[derive(Clone, Eq, PartialEq, Debug, Encode, Decode)]
 #[cfg_attr(feature = "std", derive(Serialize))]
-#[cfg_attr(feature = "std", serde(rename_all = "camelCase", tag = "type"))]
+#[cfg_attr(feature = "std", serde(rename_all = "lowercase", tag = "type"))]
```

Comment:
tag type is useless here

Reply:
Why?

Reply:
So the type field is needed for the produced json ?

Reply:
Yeah, each variant is serialized as the value for the `type` property.

Reply:
ok, my bad